examples/28_camera_video_example.py

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env python3
+
+import cv2
+import depthai as dai
+import numpy as np
+
+# Start defining a pipeline
+pipeline = dai.Pipeline()
+
+# Define a source - color camera
+colorCam = pipeline.createColorCamera()
+colorCam.setBoardSocket(dai.CameraBoardSocket.RGB)
+colorCam.setResolution(dai.ColorCameraProperties.SensorResolution.THE_1080_P)
+colorCam.setVideoSize(1920, 1080)
+
+# Create output
+xoutVideo = pipeline.createXLinkOut()
+xoutVideo.setStreamName("video")

```

Comment:
Add:
```
xoutVideo.input.setBlocking(False)
xoutVideo.input.setQueueSize(1)
```
This will drop frames before transferring if USB (USB2) is bottleneck - reduces latency in that case, as frames aren't queued up